rpc/src/rpc.rs

```diff
@@ -4938,6 +4990,20 @@ pub mod tests {
             bank.store_account(vote_pubkey, &vote_account);
         }
 
+        fn update_prioritization_fee_cache(&self, transactions: Vec<Transaction>) {
```

Comment:
Is this for tests only, or intended for other use?

Reply:
Tests only. This is inside `mod tests`, and implemented on RpcHandler, which is a test harness.